docs/install-pro.md

```diff
@@ -0,0 +1,113 @@
+# Install Percona XtraDB Cluster Pro
+
+--8<--- "pro-build-announcement.md"
+
+This document provides guidelines how to install Pro packages of Percona XtraDB Cluster from Percona repositories. [Check files in packages built for Percona XtraDB Cluster Pro :material-arrow-right:](pro-files.md){.md-button}
+
+## Prerequisites
+
```

Comment:
Add the snippet before the installation instructions.
We can't tell them enough that it's for customers.

Reply:
Added